docs/source/partial-transformers/migrating-from-lifted.rst

```diff
@@ -0,0 +1,98 @@
+.. _migrating-from-lifted:
+
+Migrating from Lifted Transformers
+==================================
+
+Chimney's :ref:`lifted-transformers` were historically first experimental attempt
+to express transformations that may potentially fail. Despite their great expressiveness, they were
+lacking several basic features and had few design flaws that didn't make them attractive for wider usage.
```

Comment:
and had a few design flaws that  make them unattractive/difficult for wider audience.